test/byContract/IexecPoco/IexecPoco1.test.ts

```diff
@@ -196,74 +179,137 @@ describe('IexecPoco1', () => {
+                trust: trust,
+                category: category,
+                params: params,
+            });
+            await depositForRequesterAndSchedulerWithDefaultPrices(botVolume);
+            // Sign and match orders.
+            const startTime = await setNextBlockTimestamp();
+            await signOrders(iexecWrapper.getDomain(), standardOrders, ordersActors);
+            const dealId = getDealId(iexecWrapper.getDomain(), standardOrders.requester);
+            await expect(iexecPocoAsRequester.matchOrders(...standardOrders.toArray())).to.emit(
+                iexecPoco,
+                'OrdersMatched',
+            );
+            // Check deal
+            const deal = await iexecPoco.viewDeal(dealId);
+            expect(deal.app.pointer).to.equal(appAddress);
+            expect(deal.app.owner).to.equal(appProvider.address);
+            expect(deal.app.price).to.equal(appPrice);
+            expect(deal.dataset.pointer).to.equal(datasetAddress);
+            expect(deal.dataset.owner).to.equal(datasetProvider.address);
+            expect(deal.dataset.price).to.equal(datasetPrice);
+            expect(deal.workerpool.pointer).to.equal(workerpoolAddress);
+            expect(deal.workerpool.owner).to.equal(scheduler.address);
+            expect(deal.workerpool.price).to.equal(workerpoolPrice);
+            expect(deal.trust).to.equal(trust);
+            expect(deal.category).to.equal(category);
+            expect(deal.tag).to.equal(standardDealTag);
+            expect(deal.requester).to.equal(requester.address);
+            expect(deal.beneficiary).to.equal(beneficiary.address);
+            expect(deal.callback).to.equal(randomAddress);
             expect(deal.params).to.equal(params);
-            expect(deal.startTime.toNumber()).to.equal(startTime);
-            expect(deal.botFirst.toNumber()).to.equal(0);
-            expect(deal.botSize.toNumber()).to.equal(volume);
-            expect(deal.workerStake.toNumber()).to.equal(workerStakePerTask);
-            expect(deal.schedulerRewardRatio.toNumber()).to.equal(schedulerRewardByTask);
+            expect(deal.startTime).to.equal(startTime);
+            expect(deal.botFirst).to.equal(0);
+            expect(deal.botSize).to.equal(botVolume);
+            expect(deal.workerStake).to.equal(
+                await iexecWrapper.computeWorkerTaskStake(workerpoolAddress, workerpoolPrice),
+            );
+            expect(deal.schedulerRewardRatio).to.equal(
+                await iexecWrapper.getSchedulerTaskRewardRatio(workerpoolAddress),
+            );
             expect(deal.sponsor).to.equal(requester.address);
         });
 
-        it('[TEE] Should match orders without callback', async () => {
-            orders.requester.callback = AddressZero;
-            await depositForRequesterAndSchedulerWithDefaultPrices();
+        it('Should match orders without beneficiary, BoT, callback, replication', async () => {
```

Comment:
Very minor, I wonder the same in my tests to avoid confusion, maybe
```suggestion
        it('Should match orders without: beneficiary, BoT, callback, replication', async () => {
```
or (no benef, no BoT, ..)?

Reply:
Why not yes.